src/orchestrator/common/util.py

```diff
@@ -275,7 +275,7 @@ def collectOutgoingMetrics(self, service_start, data_request, headers_request, r
             self.sum["outgoingTransactionResponseSize"] += len(json.dumps(data_response)) + len(str(response.headers.headers)) if 'headers' in response.headers else 0
             self.sum["serviceTimeTotal"] += (service_stop - service_start)
         except Exception, ex:
-            self.logger.error("ERROR collecting outgoing metrics %s", ex)
+            self.logger.warn("ERROR collecting outgoing metrics %s", ex)
```

Comment:
I guess this changes is so minor it doesn't deserves an entry in changelog, isn't it?

Reply:
Done